src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.minutes {
+  position: absolute;
+  bottom: 50%;
+
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+
+  animation: rotate-bottom 3600s steps(60, end) infinite forwards;
+
+  &--speed-up {
+    animation: rotate-bottom 600s steps(60, end) infinite forwards;
+  }
+}
+
+.seconds {
+  position: absolute;
+  bottom: 50%;
+
+  height: 38vmin;
+  width: 1.5vmin;
+  background-color: #2c8000;
+
+  animation: rotate-bottom 60s linear infinite;
```

Comment:
The animation for `.seconds` should also be verified to ensure it matches the intended design. If it's emulating a real stopwatch, the duration should be 60 seconds to represent one full minute.